app/config.py

```diff
@@ -72,6 +72,8 @@
 if 'DATABASE_URL' in env:
     SQLALCHEMY_DATABASE_URI = env.get('DATABASE_URL')
 
+# Authentication Max_age
```

Comment:
This comment is unnecessary. If the setting name is not expressive enough, change it.